x-pack/plugins/search_indices/public/components/indices/details_page.tsx

```diff
@@ -221,8 +228,37 @@ export const SearchIndexDetailsPage = () => {
             bottomBorder={false}
             rightSideItems={[
               <EuiFlexGroup gutterSize="m">
-                <EuiFlexItem>
-                  {!isDocumentsExists ? (
+                {isDocumentsExists ? (
+                  <>
+                    <EuiFlexItem>
+                      <EuiButtonEmpty
+                        isLoading={isDocumentsLoading}
+                        data-test-subj="useInPlaygroundLink"
```

Comment:
typo in data-test-subj

Reply:
can we also have an update in FTR to assert the presence of these actions please?